File: Python3/1061 - Tempo de um Evento.py
a1 = input().split()
dia1 = int(a1[1])
b1 = input().split()
hora1 = int(b1[0])
min1 = int(b1[2])
sec1 = int(b1[4])

a2 = input().split()
dia2 = int(a2[1])
b2 = input().split()
hora2 = int(b2[0])
min2 = int(b2[2])
sec2 = int(b2[4])

dia = dia2 - dia1
hora = hora2 - hora1
if hora < 0:
    hora += 24
    dia -= 1
min = min2 - min1
if min < 0:
    min += 60
    hora -= 1
sec = sec2 - sec1
if sec < 0:
    sec += 60
    min -= 1
if dia <= 0:
    dia = 0

print('{} dia(s)'.format(dia))
print('{} hora(s)'.format(hora))
print('{} minuto(s)'.format(min))
print('{} segundo(s)'.format(sec))

File: CSharp/1061 - Tempo de um Evento.cs
using System;

class URI {
  public static void Main (string[] args) {
    string[] S1 = Console.ReadLine().Split();
    string[] S2 = Console.ReadLine().Split();
    string[] S3 = Console.ReadLine().Split();
    string[] S4 = Console.ReadLine().Split();

    int D = int.Parse(S3[1]) - int.Parse(S1[1]);
    int H = int.Parse(S4[0]) - int.Parse(S2[0]);
    if (H < 0){
        H += 24;
        D --;}
    
    int M = int.Parse(S4[2]) - int.Parse(S2[2]);
    if (M < 0){
        M += 60;
        H --;}
    
    int S = int.Parse(S4[4]) - int.Parse(S2[4]);
    if (S < 0){
        S += 60;
        M --;}
    if (D <= 0){
        D = 0;}

    Console.WriteLine("{0} dia(s)\n{1} hora(s)\n{2} minuto(s)\n{3} segundo(s)", D, H, M, S);
  }
}